README.md

```diff
@@ -0,0 +1,116 @@
+# Grunt Email Design Workflow
+based on [github.com/leemunroe](https://github.com/leemunroe/grunt-email-workflow)
+
+## Changelog 
+
+### v0.2.0 - 09/01/2024
+
+* update packages and dependencies for current node versions: nvm not needed
+* disable third-party connections (see [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation))
+
+## Purpose
+
+Designing and testing emails is a pain. HTML tables, inline CSS, various devices and clients to test, and varying support for the latest web standards.
+
+This Grunt task helps simplify things.
+
+1. Compiles your SCSS to CSS
+2. Builds your HTML email templates
+3. Inlines your CSS
+
+## Requirements
+
+You may already have these installed on your system. If not, you'll have to install them.
+
+* Node.js - [Install Node.js](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager)
+* Grunt-cli and Grunt (`npm install grunt-cli -g`)
+* (optional) [Litmus](https://litmus.com) - Tests the email across all clients/browsers/devices
```

Comment:
We can remove the Litmus comment as no longer relevant to this repo

Reply:
Done.